cloudformation/account_resources.yml

```diff
@@ -839,6 +893,40 @@ Outputs:
     Description: ARN of the client cert secret for sandbox
     Value: !GetAtt ClientSandboxCertSecret.Id
 
+  ##################################################
+  # Clinical Tracker CA Secrets Outputs
+  ##################################################
+  ClinicalTrackerCAKeySecret:
+    Description: ARN of the clinical tracker CA key secret
+    Value: !GetAtt ClinicalTrackerCAKeySecret.Id
+    Export:
+      Name: !Join [":", [!Ref "AWS::StackName", "ClinicalTrackerCAKeySecret"]]
+
+  ClinicalTrackerCACertSecret:
+    Description: ARN of the clinical tracker CA cert secret
+    Value: !GetAtt ClinicalTrackerCACertSecret.Id
+    Export:
+      Name: !Join [":", [!Ref "AWS::StackName", "ClinicalTrackerCACertSecret"]]
+
+  ##################################################
+  # Clinical Tracker Client Secrets Outputs
+  ##################################################
+  ClinicalTrackerClientKeySecret:
+    Description: ARN of the clinical tracker client key secret
+    Value: !GetAtt ClinicalTrackerClientKeySecret.Id
+
```

Comment:
can you add an export for this as you have done for ClinicalTrackerCACertSecret

Reply:
The export section has been added.